portfolio/src/main/webapp/index.html

```diff
@@ -1,33 +1,43 @@
 <!DOCTYPE html>
 <html>
-  <head>
+<head>
     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Matthew Wong 🎯</title>
-    <link rel="shortcut icon" type="image/jpg" href="/images/Raptors.png"/>
+    <link rel="shortcut icon" type="image/jpg" href="/images/Raptors.png" />
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
-  </head>
+</head>
 
-  <body class="cursor">
+  <body>
```

Comment:
Optional: Stretch goal is to have a simple menu or content bar that lets you switch pages back and forth from the"/learnings" page easier.